src/renderers/typescript-renderer.spec.ts

```diff
@@ -620,6 +626,62 @@ describe('TypescriptModelRenderer', () => {
             `);
         });
 
+        it('outputs request body "any" on application/json body without a schema', async () => {
+            const input: ParsedMeshRAML = {
+                baseUri: '/api/v1',
+                endpoints: [{
+                    description: 'Description of the endpoint.',
+                    method: 'POST',
+                    url: '/some-endpoint',
+                    queryParameters: { },
+                    requestBody: {
+                        mimeType: 'application/json',
+                    },
+                    responses: {
+                        200: {
+                            description: 'Description of the response',
+                            responseBodySchema: {
+                                type: 'object',
+                                id: 'urn:jsonschema:com:gentics:mesh:core:rest:node:NodeResponse',
+                                properties: { } // irrelevant for this test
+                            }
+                        }
+                    }
+                }],
+                models: { },
+                version: '0.9.1'
+            };
+
+            renderer.options.endpointInterface = 'ApiEndpoints';
+            const result = await renderer.generateEndpointList(input);
+
+            expect(result).to.equal(unindent `
+                /** List of all API endpoints and their types */
+                export interface ApiEndpoints {
+                    GET: { };
+                    POST: {
+                        /** Description of the endpoint. */
+                        '/some-endpoint': {
+                            request: {
+                                urlParams?: { };
+                                queryParams?: { };
+                                body: any;
+                            };
+                            responseType: NodeResponse;
+                            responseTypes: {
+                                /** Description of the response */
+                                200: NodeResponse;
+                            };
+                        };
+                    };
+                    PATCH: { };
+                    PUT: { };
+                    DELETE: { };
+                }
+
+            `);
+        })
+
```

Comment:
Very nice addition! Glad to see my code is sufficiently manageable :-)

:+1:

Reply:
Thanks, it's a very useful library, that's why I try to rely on it as much as possible.